xbmc/games/addons/GameClientInGameSaves.cpp

```diff
@@ -26,9 +26,10 @@ using namespace GAME;
 #define INGAME_SAVES_EXTENSION_SAVE_RAM ".sav"
 #define INGAME_SAVES_EXTENSION_RTC      ".rtc"
 
-CGameClientInGameSaves::CGameClientInGameSaves(CGameClient* addon, const KodiToAddonFuncTable_Game* dllStruct) :
-  m_gameClient(addon),
-  m_dllStruct(dllStruct)
+CGameClientInGameSaves::CGameClientInGameSaves(CGameClient* addon,
+                                               const AddonInstance_Game* dllStruct)
+  : m_gameClient(addon)
+  , m_dllStruct(dllStruct)
```

Comment:
> For lines up to [line length](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#line-length) everything stays on one line, excluding the braces which must be on the following lines.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#116-constructor-initialzation-lists

Reply:
I can change to new with "comma" behind, but to have in one line make for me only sense if only one value, with more and on changes is more hard to see what is changed,

Reply:
The decision was mad that multiple values also go into one line.
You even approved that coding style change.

Reply:
My approval was related to the example with:
```cpp
CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) : m_iHiddenChannels(0)
{
}
```
and only one value, thats why it was acceptable for me.

Reply:
https://github.com/xbmc/xbmc/compare/35b2594f369a8b79f22471943a4c47d526023dbc...b3eea1f94c0612f39eb49cfb510122ed9485547a was the state of the PR at the time you approved it and it contained already the following.

### 11.6. Constructor Initialzation Lists
For short lines everything on one line is OK, excluding the braces which must be on the following lines.
```cpp
MyClass::CMyClass(bool bBoolArg, int iIntegerArg) : m_bArg(bBoolArg), m_iArg(iIntegerArg)
{
}
```

Reply:
I change it :smile:, but with `For short lines everything on one line is OK` sounds for me more like as **"optional"** :wink:.

Reply:
This change is still missing.

Reply:
See my note there https://github.com/xbmc/xbmc/pull/16701#issuecomment-544144978
Also is for me the guideline more by a one line and not like this:
```cpp
CGameClientInGameSaves::CGameClientInGameSaves(CGameClient* addon,
                                               const AddonInstance_Game* dllStruct)
  : m_gameClient(addon), m_dllStruct(dllStruct)
{
}
```